cli/cli/out/hook.go

```diff
@@ -0,0 +1,28 @@
+package out
+
+import (
+	"io"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// Hook is a hook that writes logs of specified LogLevels to specified Writer
+type Hook struct {
+	Writer    io.Writer
+	LogLevels []log.Level
+	Formatter log.Formatter
+}
```

Comment:
you could add a constructor for this object and then make the fields private maybe